### PR TITLE
add wheelbuilder images for python 3.7-9

### PR DESCRIPTION
**Pull request by renefritze**

### CODE REVIEW

File: debian/include/full [hunk 1]
```diff
@@ -11,7 +11,7 @@ RUN APT_UPDATE APT_REDIRECT && \
     APT_INSTALL doxygen LIBCPP_DEV clang-${CLANG_VERSION} \
         libeigen3-dev libscotchmetis-dev libscotchparmetis-dev libsuitesparse-dev \
         libmumps-scotch-dev ${CLANG_PYTHON_PACKAGE}-${CLANG_VERSION} lcov curl \
-        ninja-build libopenmpi-dev openmpi-common openmpi-bin libglu1-mesa libqt5x11extras5 \
```
Comment (ftalbrecht) [inline debian/include/full:14]: If I recall correctly, these are required to avoid errors (or warnings) in matplotlib visualization in Jupyter notebooks.

Comment (renefritze) [reply debian/include/full:14]: We don't have to worry actually: https://github.com/dune-community/Dockerfiles/pull/105/commits/c82bfa0a2fc618b6955e778732de7a8f63646e9f

**Comment by renefritze** [comment]
The ubuntu font you got installed in the qtcreator image @ftalbrecht is no longer available under current url. Also, new version is empty, only holds a changelog. I'm installing https://packages.ubuntu.com/hirsute/all/fonts-ubuntu/filelist instead which looks like it has all the ttfs the other package also had.

**Comment by ftalbrecht** [comment]
> The ubuntu font you got installed in the qtcreator image @ftalbrecht is no longer available under current url. Also, new version is empty, only holds a changelog. I'm installing https://packages.ubuntu.com/hirsute/all/fonts-ubuntu/filelist instead which looks like it has all the ttfs the other package also had.

thx

**Pull request merged**